tools/tests/form-control/validation.test.js

```diff
@@ -0,0 +1,169 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const errorMessages = new Map([
+    ['notAForm', 'This element is not part of a form.'],
+    ['tooLong', 'The value is too long. Maximum length is 5.'],
+    ['tooShort', 'The value is too short. Minimal length is 3.'],
+    ['rangeOverflow', 'The value is too big. Maximum is 30.'],
+    ['rangeUnderflow', 'The value is too small. Minimum is 10.'],
+    ['valueMissing', 'The value is required.'],
+    ['nameMissing', 'The elements does not have a name attribute and will not be submitted'],
+]);
+
+
+function getTextContent(element) {
+    return element.textContent.replace(/(\n)+/g, '').trim();
+}
+
+
+function setupFormValidationTestPage() {
+    const template = `
+    <gameface-form-control>
+        <input id="role" name="role" value="some name" required />
+        <input minimum="3" maximum="5" name="username" value="Valid" type="text" id="username" />
+        <input min="10" max="30" value="15" name="password" type="text" id="password" />

```

Comment:
```suggestion
        <input minlength="10" maxlength="30" value="15" name="password" type="text" id="password" />
```

Reply:
There are the [min and max attributes of a number input](https://www.w3schools.com/tags/att_input_min.asp).
I'm preparing the field for the textField validation.

Reply:
We will add the test cases for the text field when its validation is ready. Until then we can probably add just test cases that are related to the current validation.